src/modules/services/redis.nix

```diff
@@ -67,14 +74,15 @@ in
       cfg.package
     ];
 
+    env.REDIS_UNIX_SOCKET = REDIS_UNIX_SOCKET;
```

Comment:
I wonder if we should set this on non-zero port?

Reply:
Yeah, you're right. It is only needed when port is zero.

I don't know what the right way to conditionally set an environment variable would be, would this work.

```nix
env.REDIS_UNIX_SOCKET = if cfg.port == 0 then REDIS_UNIX_SOCKET else null;
```

Reply:
I went with `optionalAttrs` instead of setting it to `null`.